cider-velox/src/CiderVeloxPluginCtx.cpp

```diff
@@ -20,14 +20,14 @@
  */
 
 #include "CiderVeloxPluginCtx.h"
-#include <mutex>
 #include "CiderPlanNodeTranslator.h"
 #include "exec/plan/parser/ConverterHelper.h"
 
 namespace facebook::velox::plugin {
 void facebook::velox::plugin::CiderVeloxPluginCtx::init() {
   registerTranslator();
   registerVeloxExtensionFunction();
+  ciderTransformerFactory_.registerCiderPattern();
```

Comment:
facebook::velox::plugin::CiderVeloxPluginCtx::init is called only once
so I think, registerCiderPattern should be defined as a global function, not a member method.